docs/spec/lightclient/verification/Blockchain_A_1.tla

```diff
@@ -110,7 +110,7 @@ FaultAssumption(pFaultyNodes, pNow, pBlockchain) ==
 (* Can a block be produced by a correct peer, or an authenticated Byzantine peer *)
 IsLightBlockAllowedByDigitalSignatures(ht, block) == 
     \/ block.header = blockchain[ht] \* signed by correct and faulty (maybe)
-    \/ block.Commits \subseteq Faulty /\ block.header.height = ht \* signed only by faulty
+    \/ block.Commits \subseteq Faulty /\ block.header.height = ht /\ block.header.time > 0 \* signed only by faulty
```

Comment:
I guess faulty processes may be allowed to write any time, also negative. Or is the time stored in some unsigned domain in the implementation?

Reply:
it is indeed stored and serialized/deserialized as an unsigned integer, so signed integers do no make sense and only add confusion, I am afraid

Reply:
I guess in the English spec we implicitly also assume positive times. Perhaps we should clarify it there explicitly.